iredis/entry.py

```diff
@@ -160,8 +160,7 @@ def __next__(self):
 
 
 def prompt_message(client):
-    # TODO custom prompt
-    text = "{hostname}> ".format(hostname=str(client))
```

Comment:
I moved `> ` into the `__str__` function.

Reply:
yes, we should do this